packages/core/admin/utils/get-custom-webpack-config.js

```diff
@@ -8,13 +8,18 @@ const fs = require('fs-extra');
 const getWebpackConfig = require('../webpack.config');
 
 const getCustomWebpackConfig = (dir, config) => {
-  const adminConfigPath = path.join(dir, 'src', 'admin', 'webpack.config.js');
+  const adminConfigPathJS = path.join(dir, 'src', 'admin', 'webpack.config.js');
+  const adminConfigPathTS = path.join(dir, 'src', 'admin', 'webpack.config.ts');
 
   let webpackConfig = getWebpackConfig(config);
 
-  if (fs.existsSync(adminConfigPath)) {
-    const webpackAdminConfig = require(path.resolve(adminConfigPath));
-
+  if (fs.existsSync(adminConfigPathJS) || fs.existsSync(adminConfigPathTS)) {
```

Comment:
Could you extract those `fs.existsSync` calls to some variables so that you don't need to recompute it a second time on line 18?